helpers/uniqueInts_test.go

```diff
@@ -0,0 +1,60 @@
+package helpers
+
+import (
+	"sort"
+	"testing"
+)
+
+func TestUniqueInts(t *testing.T) {
+	m := make(UniqueInts)
+	a := m.Array()
+	if m.Array() == nil {
+		t.Fatalf("uniqueInts array was nil")
+	}
+	validateIntArray(t, "uniqueInts", a, []int{})
+
+	m.AddValue(5)
+	m.AddValue(5)
+	m.AddValue(22)
+	a = m.Array()
+	if a == nil {
+		t.Fatalf("uniqueInts array was nil")
+	}
+
+	validateIntArraySortable(t, "uniqueInts", a, []int{5, 22}, true)
+
+	if !m.HasValue(5) {
+		t.Error("uniqueInts collection doesn't contain expected value: 5")
+	}
+
+	if m.HasValue(10) {
```

Comment:
that seems.. specific?

Reply:
old file that was moved, I think it's just a test to make sure it doesn't always return true.

Reply:
i.e. value doesn't matter